tests/wpunit/Tribe/Tickets/Commerce/RSVP/__snapshots__/RSVPBlockTest__test_should_render_ticket_block with data set 0__1.php

```diff
@@ -44,9 +44,9 @@ class="tribe-block__rsvp__status-button tribe-block__rsvp__status-button--going"
 			</div>
 				<div class="tribe-common">
 	<div  class="tribe-tickets-loader__dots tribe-common-c-loader tribe-common-a11y-hidden tribe-block__rsvp__loading"  >
-		<div class="tribe-common-c-loader__dot tribe-common-c-loader__dot--first"></div>
-		<div class="tribe-common-c-loader__dot tribe-common-c-loader__dot--second"></div>
-		<div class="tribe-common-c-loader__dot tribe-common-c-loader__dot--third"></div>
+		
+		
+		
```

Comment:
I'm wondering why this one doesn't have the new dots?

Reply:
The new snapshot has SVGs but the SVGs are preg replace removed by the snapshot code due to a separate SVG-related issue.